setup.py

```diff
@@ -18,7 +18,7 @@
     author_email='kootenpv@gmail.com',
     license='MIT',
     extras_require={"all": ["keyring"]},
-    install_requires=["premailer"],
+    install_requires=["premailer", "dkimpy"],
```

Comment:
I think we better make it an extras require as I don't think this should be there by default.

```
-    extras_require={"all": ["keyring"]},
+    extras_require={"all": ["keyring", "dkim"], "dkim": ["dkim"]},
```

I think we can just try to 

```python
try:
    import dkim
except ImportError:
   pass
```

In the readme we can show how to install:

    pip install yagmail[dkim]

Reply:
ok, good idea, will add